pkgs/os-specific/linux/rtl8852bu/default.nix

```
@@ -30,7 +30,9 @@ stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace ./Makefile \
       --replace-fail /sbin/depmod \# \
-      --replace-fail '$(MODDESTDIR)' "$out/lib/modules/${kernel.modDirVersion}/kernel/net/wireless/"
+      --replace-fail '$(MODDESTDIR)' "$out/lib/modules/${kernel.modDirVersion}/kernel/net/wireless/" \
+      --replace-fail 'cp -f $(MODULE_NAME).conf /etc/modprobe.d' "" \
```

IMO it would be more useful to copy this to the output, what do you think?

```suggestion
      --replace-fail 'cp -f $(MODULE_NAME).conf /etc/modprobe.d' "cp -f 8852bu.conf ${placeholder "out"}/etc/modprobe.d" \
```